packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -327,9 +323,7 @@ const GlobalHeader = ({ className }) => {
                 size={Button.SIZE.EXTRA_SMALL}
                 variant={Button.VARIANT.PRIMARY}
               >
-                <span>
-                  {inDocsSite ? t('button.signUp') : t('button.startNow')}
-                </span>
+                <span>{'Free Account'}</span>
```

Comment:
We should make sure this ends up in our translation file. Rather than using the raw text `Free Account` here (which also should be sentence case), case you instead use `button.signUp` and change the translation in `src/i18n/translations/en.json`?

Reply:
Oh, and we should also be able to delete the `button.startNow` value in `en.json`

Reply:
will do